Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md

```diff
@@ -0,0 +1,97 @@
+
+Please refer to the [help documentation](https://help.boldbi.com/embedding-options/embedding-sdk/samples/asp-net-core/#how-to-run-the-sample) to know how to run the sample.
+
+## How to apply dashboard parameters for widget in JavaScript Embedding
+
+JavaScript embedding provides support for filtering widgets with dashboard parameters. This section will elaborate on how to effectively utilize dashboard parameters in JavaScript embedding to filter the widgets.
+
+### Steps to follow for applying filters into widgets,
+1. In the Bold BI Server, load the `HR Recruitment Dashboard` from Sample Dashboards.
+
+    ![load-dashboard-from-server](https://github.com/boldbi/samples/assets/92368448/fa007d90-4b55-4d42-9562-9cb120876f4d)
+
+2. By default, the `HR Recruitment Dashboard` does not have [Dashboard Parameters](https://help.boldbi.com/cloud-bi/working-with-data-source/configuring-dashboard-parameters/#configuring-dashboard-parameters) configured. For demonstration, follow these steps to configure the Dashboard Parameter with `Literal Mode` as shown in the screenshot below:
+
+    * Enter the Parameter Name: `Department`
+    * Enter the Parameter Value: `IT`
+
+    ![enter-parameter-name-value](https://github.com/boldbi/samples/assets/92368448/d4041b0a-517a-4c3d-abf7-c05e25615630)
+
+3. After successfully publishing the dashboard with configured dashboard parameters, download the [embedConfig JSON](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) file associated with that specific dashboard. 
+
+4. Once configured the embedConfig.JSON file in the designated [location](https://github.com/boldbi/aspnet-core-sample/tree/master/BoldBI.Embed.Sample), then run your application.
+
+5. Upon initial rendering, the widget will display with the default value of the dashboard parameter, which, in this case, is set to `IT`. The property panel, by default, will show the name of the dashboard parameter in the text field. 
+
+      ![initial-widget-rendering](https://github.com/boldbi/samples/assets/92368448/14b07a07-0f18-42e8-a130-f85a092ffffa)
+
+>**Note:** For more information about the initial rendering of the widget, please refer the [link](https://help.boldbi.com/embedding-options/embedding-sdk/embedding-a-widget/).
+
+6.  To modify the widget according to a particular dashboard parameter value, choose a different value from the list of `Departments`, then click on `Apply`. For example, select `Accounts` to demonstrate.
+
+    ![select-department-values](https://github.com/boldbi/samples/assets/92368448/6003e0dc-216f-4654-9217-4ff76cac32d0)
+
+>**Note:** For more information about updating the widget with dashboard parameter, please refer the [link](https://help.boldbi.com/embedding-options/embedding-sdk/embedding-api-reference/methods/#updatefilters).
```

Comment:
For more information about updating the widget with dashboard parameter without re-render the widget.

Reply:
Resolved the concern and changed the content